### PR TITLE
truncate long event description

### PR DESCRIPTION
**Pull request by sumitvekariya**
## Description

The event description had no text limitation and could have been too long. So now event description is limited to 230 characters, if longer than that it is truncated to 230 characters followed by `...`

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/13914

### CODE REVIEW

File: src/components/EventCard.tsx [hunk 1]
```diff
@@ -40,6 +40,12 @@ const EventCard = ({
     new Date(endDate?.replace(/-/g, "/"))
   )
 
+  const MAX_DESCRIPTION_CHARS = 230
+  const truncatedDescription =
+    description.length > MAX_DESCRIPTION_CHARS
+      ? `${description.slice(0, MAX_DESCRIPTION_CHARS)}...`
+      : description
+
```
Comment (wackerow) [inline src/components/EventCard.tsx:48]:
```suggestion
```

File: src/components/EventCard.tsx [hunk 2]
```diff
@@ -108,7 +114,7 @@ const EventCard = ({
               </Text>
             </Box>
             <Box>
-              <Text fontSize="sm">{description}</Text>
+              <Text fontSize="sm">{truncatedDescription}</Text>
```
Comment (wackerow) [inline src/components/EventCard.tsx:117]: We'll just migrate this specific element so we can use tailwind:
```suggestion
              <p className="text-sm line-clamp-6">{description}</p>
```

**Review by wackerow — CHANGES_REQUESTED** [review]
Hey @sumitvekariya! Thanks for the PR... Apologies if the issue may have been misleading, but I think we can get away with simply restricting the _line_ count here (instead of _word_ count), which we can use a simply TailwindCSS class for this: `line-clamp-x`... by default this has up to `line-clamp-6`, so I would suggest this )(just chatted with @konopkja as well who is in agreement)

Left a suggestion for how we can simplify to use this approach

Screenshot2024_09_23_090716

**Review by corwintines — APPROVED** [review]
Thanks @sumitvekariya!

**Pull request merged**